messaging/kafka-producer/src/test/java/io/quarkus/ts/messaging/kafka/producer/SnappyCompressionIT.java

```diff
@@ -0,0 +1,162 @@
+package io.quarkus.ts.messaging.kafka.producer;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import java.time.Duration;
+import java.util.Collections;
+import java.util.Properties;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.apache.kafka.clients.consumer.ConsumerConfig;
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+import org.apache.kafka.clients.consumer.KafkaConsumer;
+import org.apache.kafka.common.serialization.IntegerDeserializer;
+import org.apache.kafka.common.serialization.StringDeserializer;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.testcontainers.containers.GenericContainer;
+
+import io.quarkus.test.bootstrap.KafkaService;
+import io.quarkus.test.bootstrap.Protocol;
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.services.KafkaContainer;
+import io.quarkus.test.services.QuarkusApplication;
+import io.quarkus.test.services.containers.DockerContainerManagedResource;
+import io.quarkus.test.services.containers.model.KafkaVendor;
+import io.smallrye.mutiny.Uni;
+import io.smallrye.mutiny.helpers.test.UniAssertSubscriber;
+import io.vertx.mutiny.core.buffer.Buffer;
+
+@QuarkusScenario
+public class SnappyCompressionIT {
+
+    private static final int TIMEOUT_SEC = 5;
+
+    private static final String BIG_JSON = "/big_json.json";
+
+    private static final String FILTER_COMMAND_LOG_CONTAINER = "/bin/bash -c \"./bin/kafka-run-class.sh kafka.tools.DumpLogSegments --deep-iteration --print-data-log --files /tmp/kafka-logs/test-0/00000000000000000000.log | head\"";
+    private static final String[] COMMAND_LOG_CONTAINER = { "/bin/bash", "-c", FILTER_COMMAND_LOG_CONTAINER };
```

Comment:
This is weird. You have one command starting with "/bin/bash -c" wrapping around another command, which starts with "/bin/bash -c".
Also these two commands can be IMHO simplified into one. As far as I see FILTER_COMMAND_LOG_CONTAINER is not used on its own anyway.